epub33/explainers/EPUB33-explainer.md

```diff
@@ -0,0 +1,114 @@
+
+1. Tracking reading position. You really do want the reading system to remember that you just started chapter 5. 
+2. Remembering highlights, notes, bookmarks. 
+3. Remembering user preferences around fonts, visual themes (including dark/light), text justification, etc. 
+4. Recording progress through a book and whether a book was finished. 
+5. I am aware of some reading systems that track every reading system, including what tie of day, the duration, how many pages read, what book, IP address, etc. 
+
+
+
+### 2. Is this specification exposing the minimum amount of information necessary to power the feature?
+
+The EPUB specification does not mention reading system features like bookmarks or highlights. There is no requirement to remember reading state. 
+
+### 3. How does this specification deal with personal information or personally-identifiable information or information derived thereof?
+
+??
+
+### 4. How does this specification deal with sensitive information?
+
+Merely the fact that you are reading a particular book can be sensitive information. Again, the specification offers no guidance to reading systems on this matter. 
+
+### 5. Does this specification introduce new state for an origin that persists across browsing sessions?
+
+Reading systems, in order to meet user expectations, must retain some information across browsing sessions. The spec does not mention this, as it does not mandate this user agent behavior. 
+
+### 6. What information from the underlying platform, e.g. configuration data, is exposed by this specification to an origin?
+
+Many EPUB reading systems do not support scripting at all. For those that do, scripts in EPUBs that seek this sort of information might be blocked by the application before they even reach the platform level. If not, the risks are likely similar to that of HTML/CSS. 
+
+
+### 7. Does this specification allow an origin access to sensors on a user’s device?
+
+EPUB does have requirements for user agents to respond to device orientation (`portrait | landscape`) for fixed-layout EPUBs. 
+
+There has been talk about scripting EPUB based on geolocation information; I don't know if this has happened in the wild yet, or if any reading systems would allow access to this information.
+
+### 8. What data does this specification expose to an origin? Please also document what data is identical to data exposed by other features, in the same or different contexts.
+
+??
+
+### 9. Does this specification enable new script execution/loading mechanisms?
+
+Not to my knowledge. Scripting is not forbidden, but follows the HTML model.
+
+### 10. Does this specification allow an origin to access other devices?
+
+EPUB reading systems often implement syncing behaviour across multiple devices with the same user account. This is not required or mentioned by the spec. 
+
```

Comment:
Isn't again the point that we define what our origin is, which has consequences for such questions (and whose answer does not depend on this spec)?